<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtdViewerMain</class>
 <widget class="QMainWindow" name="OtdViewerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OtdViewerMain</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Connections.bmp</normaloff>Connections.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_6"/>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="monitor">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>10</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximumBlockCount">
       <number>2048</number>
      </property>
      <property name="backgroundVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Подключение</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>порт</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="otd_server_port">
           <property name="text">
            <string>45777</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>сервер</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="otd_server_addr">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="bConnect">
           <property name="text">
            <string>Подключить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Connections.bmp</normaloff>Connections.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="bClearMon">
           <property name="text">
            <string>Очистить монитор</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="b_paused">
           <property name="text">
            <string>Пауза</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>ОТД</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="cbDiag">
              <property name="text">
               <string>Диагностика</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cbData">
              <property name="text">
               <string>Данные</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="cbTimestamp">
              <property name="text">
               <string>Метка времени</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="cbPersonalDiag">
              <property name="text">
               <string>Персональная диагностика</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QCheckBox" name="cbChangesMask">
              <property name="text">
               <string>Маска изменений</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cbNames">
              <property name="text">
               <string>Имена</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="cbPartData">
              <property name="text">
               <string>Частичные данные</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="bCallData">
              <property name="text">
               <string>Вызов данных</string>
              </property>
              <property name="autoRepeat">
               <bool>true</bool>
              </property>
              <property name="autoRepeatDelay">
               <number>250</number>
              </property>
              <property name="autoRepeatInterval">
               <number>0</number>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="bOrderData">
              <property name="text">
               <string>Подписка</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="bUnorderData">
              <property name="text">
               <string>Отказ от подписки</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Адрес</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="otd_addr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>255.255.255.255</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>OTD архивы</string>
         </attribute>
         <widget class="QListView" name="ArchList">
          <property name="geometry">
           <rect>
            <x>141</x>
            <y>10</y>
            <width>481</width>
            <height>111</height>
           </rect>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::Batched</enum>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>2</x>
            <y>3</y>
            <width>138</width>
            <height>124</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QPushButton" name="bGetArchList">
             <property name="text">
              <string>Список</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGetArchive">
             <property name="text">
              <string>Получить архив</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGetAndErase">
             <property name="text">
              <string>Получить и удалить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bEraseArchive">
             <property name="text">
              <string>Удалить архив</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Аварийные уставки</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>335</width>
            <height>117</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0" rowspan="3">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Команда</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QRadioButton" name="acClear">
                  <property name="text">
                   <string>Очистить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="acLess">
                  <property name="text">
                   <string>Меньше</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="acMore">
                  <property name="text">
                   <string>Больше</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Адрес параметра</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="apAddrParam">
             <property name="text">
              <string>1.1.0.0-1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Значение</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="apValue"/>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btnSendAlarmParam">
             <property name="text">
              <string>Выполнить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>СPC152</string>
         </attribute>
         <widget class="QListView" name="cpc152_alarm_list">
          <property name="geometry">
           <rect>
            <x>132</x>
            <y>0</y>
            <width>270</width>
            <height>111</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>1</y>
            <width>131</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="bCpc152GetAlarmList">
             <property name="text">
              <string>Список</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCpc152GetAlarm">
             <property name="text">
              <string>Получить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbEraseAlarmFile">
             <property name="text">
              <string>Удалять арихвы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbOnlyArchShow">
             <property name="text">
              <string>Только архивы</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>405</x>
            <y>3</y>
            <width>74</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Устройство</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>группа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>параметр</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>478</x>
            <y>3</y>
            <width>127</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="dev_num"/>
           </item>
           <item>
            <widget class="QLineEdit" name="grp_num"/>
           </item>
           <item>
            <widget class="QLineEdit" name="param_num"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>406</x>
            <y>90</y>
            <width>190</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="RB_cpc152_off">
             <property name="text">
              <string>ОТКЛ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RB_cpc152_on">
             <property name="text">
              <string>ВКЛ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>1</y>
            <width>80</width>
            <height>66</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="bCpc152AlarmAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bCpc152AlarmErase">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bCpc152AlarmClear">
           <property name="text">
            <string>Очистить</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="act_about"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>Завершить</string>
   </property>
   <property name="toolTip">
    <string>Завершить работу</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbPersonalDiag</tabstop>
  <tabstop>cbChangesMask</tabstop>
  <tabstop>otd_server_addr</tabstop>
  <tabstop>otd_server_port</tabstop>
  <tabstop>cbDiag</tabstop>
  <tabstop>cbData</tabstop>
  <tabstop>cbTimestamp</tabstop>
  <tabstop>cbNames</tabstop>
  <tabstop>cbPartData</tabstop>
  <tabstop>bCallData</tabstop>
  <tabstop>bOrderData</tabstop>
  <tabstop>bUnorderData</tabstop>
  <tabstop>otd_addr</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
